src/python/bot/fuzzers/libFuzzer/stats.py

```diff
@@ -202,7 +203,10 @@ def parse_performance_features(log_lines, strategies, arguments):
 
   # Initialize all strategy stats as disabled by default.
   for strategy_type in strategy.LIBFUZZER_STRATEGY_LIST:
-    stats[strategy_column_name(strategy_type.name)] = 0
+    if strategy_type.name == 'peach_grammar_mutation':
```

Comment:
This is a bit too hacky.  Let's use STRATEGIES_WITH_PREFIX_VALUE_TYPE here to decide the default value.

Reply:
Let me know if this is what you had in mind. The other way I considered doing it was to store the default value of the prefix in the STRATEGIES_WITH_PREFIX_VALUE_TYPE dict as well. Which would you prefer?